XMonad/Hooks/WorkspaceHistory.hs

```diff
@@ -14,11 +14,11 @@
 --
 -----------------------------------------------------------------------------
 
-module XMonad.Hooks.WorkspaceHistory
+module XMonad.Hooks.WorkspaceHistory (
```

Comment:
Lets be consistent about how we indent thing.

Either put the parenthesis on a newline in both places or leave it here in both places.